flagsmith/models.go

```diff
@@ -226,6 +224,9 @@ type FeatureResourceData struct {
 
 func (f *FeatureResourceData) ToClientFeature() *flagsmithapi.Feature {
 	typeValue := f.Type.ValueString()
+	if typeValue == "" {
+		typeValue = "STANDARD"
```

Comment:
I'm not entirely sure why we need this if we're now able to use defaults in terraform?

Reply:
Additionally, we can add a `StringInSlice` validator to account for the lack of enum support.

See a good [example](https://github.com/hashicorp/terraform-plugin-sdk/issues/79) of this.

Reply:
> I'm not entirely sure why we need this if we're now able to use defaults in terraform?

Yep, this is the fun part, the default value is not available in the state that provider has access to. It's more or less only used in the plan that is shown to the user

Reply:
> Additionally, we can add a `StringInSlice` validator to account for the lack of enum support.
> 
> See a good [example](https://github.com/hashicorp/terraform-plugin-sdk/issues/79) of this.

Yeah, good shout